_includes/utilities/responsive-variants.html

```diff
@@ -63,21 +63,16 @@ <h4 class="font-lang-2xs margin-top-2 margin-bottom-05">Output</h4>
     </tbody>
   </table>
 
-  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings">USWDS settings configuration</a>. When modifying any settings stored as a map, you need to set <em>all</em> elements of the map, not just the element you wish to change.</p>
+  <p class="utility-text margin-top-4">Set whether a utility family outputs with responsive variants by setting the value of <code>$[utility_family]-settings.responsive</code> in your <a href="{{ site.baseurl }}/documentation/settings">USWDS settings configuration</a>. When modifying any settings stored as a map, undefined elements of the map will take its default value.</p>
 
-  <div class="example border-left-05 border-secondary-light padding-left-105 margin-top-2 margin-bottom-2">
+  <div class="example border-left-05 border-secondary-light padding-left-105 margin-top-2 margin-bottom-2 bg-white">
+    <h4 class="font-lang-2xs margin-top-0 margin-bottom-05">Example</h4>
     <pre class=" font-mono-xs margin-0 padding-0 bg-transparent">
 @use "uswds-core" with (
   $background-color-settings: (
-      output: true,
-      responsive: false,
-      active: false,
-      focus: false,
-      hover: true,
-      visited: false,
-    )
+    responsive: true
+  )
 );
```

Comment:
There is some funkiness on the page here. I believe it should be fixed by restoring this closing tag. 
```suggestion
);
</pre>
```

The funkiness:
![image](https://github.com/uswds/uswds-site/assets/93996430/e9c10cdb-ec1a-4dfb-91fa-51e6d7ebf05a)

Reply:
Resolved in [e88e606](https://github.com/uswds/uswds-site/pull/2077/commits/e88e6069e09ec8d7c3d6944bf4e0b699e8761872)